### DISCUSSION TITLE
cpg0029-chroma-pilot

### DISCUSSION THREAD

**Discussion by shntnu** [broadinstitute/cellpainting-gallery#54] [Dataset descriptions]
Profile/data repo = [Link to profile/data repo]  
Publication/analysis repo = [Link to publication/analysis repo]  
Imaging Platform project tag = 2023_08_31_Chromalive
Institution identifier = [e.g. `broad`,`anonymous`]
Imaging Platform internal "champion" = Erin

**Workflow**
- [ ] Maintainer adds dataset identifier to [prefixes](https://github.com/broadinstitute/cellpainting-gallery/blob/main/documentation/prefixes.md) document.
- [ ] Champion or Contributor fills out the metadata collection form (https://airtable.com/shrVxz9DcoMlDoCBI)
- [ ] Champion checks that metadata completely filled out in Project Profiler Database (Imaging Platform internal use only)
- [ ] Champion emails Ank with the prefix so that he can 1) open the prefix for transfer to `staging-cellpainting-gallery` and 2) create credentials that allow upload. Each credential is for a single prefix on the staging bucket; multiple people can assume that same credential.

**Transfer data to `staging-cellpainting-gallery`**
Instructions for using the staging bucket credentials are [here](https://broadinstitute.github.io/cellpainting-gallery/uploading_to_cpg.html).
For each data subtype, Champion marks off as complete or explain why it is not being included:
- [ ] Image acquisition complete and uploaded (`/images`)
- [ ] Feature extraction complete and uploaded (`/workspace/analysis`)
- [ ] Backend complete and uploaded (`/workspace/backend`)
- [ ] Load_data complete and uploaded (`/workspace/load_data_csv`)
- [ ] Profiling complete and uploaded (`/workspace/profiles`)
- [ ] Metadata complete and uploaded (`/workspace/metadata`)

**Project completion** 
These are the final steps to make a dataset fully public:
- [ ] Maintainer or champion Run [validation script](http://github.com/broadinstitute/cpg/cpgdata) to ensure data is completely uploaded (When script is out of alpha)
- [ ] Maintainer or champion updates cellpainting-gallery/README.md to add dataset to README.
- [ ] Maintainer transfers from `staging-cellpainting-gallery` to `cellpainting-gallery`.
- [ ] Ank disable staging prefix, delete data from staging, delete upload credentials.

Optional preparation for publication:
- [ ] Run Distributed-BioFormats2Raw to create .ome.zarr files
- [ ] Upload (meta)data to IDR (images remain hosted in cellpainting-gallery) 
- [ ] Make IDR entry public

Once a paper is published:
- [ ] Maintainer or champion updates [Publications](https://github.com/broadinstitute/cellpainting-gallery/docs/publications.md) (In PR by data generator)
- [ ] Contributor adds the paper to https://github.com/cytodata/awesome-cytodata

**Comment by ErinWeisbart** [comment]
One suggestion about naming of this dataset, is this only ever going to include pilot data? If so, then `chroma-pilot` is appropriate, but we have found with other datasets that started as an upload of the pilot data that then included production/published data, we needed to rename to remove the `pilot` suffix. So perhaps better to start with just `chroma`?

**Reply by shntnu** [reply]
It was named `chroma` at first but Alán said this:
 
> May I suggest adding something suggesting it is an explorative approach? (e.g,. explorechroma), Just in case it is used for other projects in the future.

I'd agree with Alán here: if end up with multiple chroma datasets, we'd want to have labeled this one as a pilot (this is different from Varchamp, where there is only one)

**Comment by bethac07** [comment]
Thanks @ErinWeisbart and @sugan89 for getting this uploaded! 

Can we add maybe a minimal readme file with what is in each batch in terms of what dyes are present in each plate? I think that's all we need to do to make this "currently usable".

**Reply by sugan89** [reply]
Hi Beth!
Sorry for not adding it earlier. 
[DetailsOfBatchesCPG.txt](https://github.com/broadinstitute/cellpainting-gallery/files/14361420/DetailsOfBatchesCPG.txt)
Please let me know if the attached information will be sufficient for the readme file. Thanks!

**Reply by bethac07** [reply]
That looks perfect! Great job.

**Comment by ErinWeisbart** [comment]
Copying Suganya's README text here for quick browse ability:
The images are for comparison of different set of dyes in the cell painting assay. Each batch has been treated with same set of compounds but are stained with different dyes. 

**Batch1:**
BR00122250 - The plate has been treated with standard cell painting dyes (Perkin Elmer)
* Hoechst 405
* ConcavalinA 488
* Nucleic acid 512 
* WGA 555
* Phalloidin 568 
* Mitotracker 647 
Contains images from 9 fields in a single well


BR00122246 - The plate has been stained with Tocris MitoBrilliant in place of MitoTracker. Rest of the dyes were from Perkin Elmer 
* Hoechst 405
* Concavalin A 488 
* Nucleic acid 512 
* WGA 555
* Phalloidin 568 
* Tocris MitoBrilliant 647 
Contains images from 9 fields in single well

**Batch2:**
BR00122247 - This plate has been stained with Phalloidin 400LS (long stoke shifted) in place of Phalloidin. 
* Hoechst 405
* Phalloidin 400LS 
* ConcavalinA 488 
* Nucleic acid 512 
* WGA 555
* Mito 647 
Contains images from 9 fields in a single well. 

**Batch3:**
BR00122248 - This batch contains the images of standard cell painting dyes. The cells were intially stained with Saguaro dyes, imaged live(these images exist in Batch6), fixed and later stained with standard cell painting dyes. 
* Hoechst 405
* ConcavalinA 488
* Nucleic acid 512 
* WGA 555
* Phalloidin 568 
* Mitotracker 647
Contains images from 9 fields in a single well. 

**Batch4:**
BR00122245 - Images are from live cells. This batch is stained with the following, 
* DRAQ7 
* Cas 3/7
Contains images from 9 fields in a single well. 

**Batch5:**
BR00122249 - Images are from live cells. This batch has been stained with Saguaro dyes 
* ChromaLive 488 Yellow (CL488Y)
* ChromaLive 561 Yellow (CL561Y) Ex561, Em570-630
* ChromaLive 488 Red (CL488R) Ex488, Em630-750 
Contains images from 9 fields in a single well.

**Batch 6, 7 and 8 contains images taken from live cells at different time points.** 

**Batch6:**
This batch contains images from three time points and following are the dyes used,

Saguaro 
* ChromaLive 488 Yellow (CL488Y)
* ChromaLive 561 Yellow (CL561Y) Ex561, Em570-630
* ChromaLive 488 Red (CL488R) Ex488, Em630-750
* DRAQ7 
* Cas3/7 

04h Live - BR00122248__2023-03-22T19_03_51-Measurement1 - contains images from 4 fields
24h Live - BR00122248__2023-03-23T17_11_00-Measurement2 - contains images from 4 fields
48h Live - BR00122248__2023-03-24T12_49_58-Measurement3 - contains images from 1 field

**Batch7:**
This batch contains from images three time points and following are the dyes used,

* DRAQ7 
* Cas3/7 

04h Live - BR00122245__2023-03-22T22_13_07-Measurement1 - contains images from 4 fields
24h Live - BR00122245__2023-03-23T15_57_45-Measurement2 - contains images from 4 fields 
48h Live - BR00122245__2023-03-24T12_32_25-Measurement3 - contains images from 1 field

**Batch8:**
This batch contains from images three time points and following are the dyes used,

Saguaro dyes 
* ChromaLive 488 Yellow (CL488Y)
* ChromaLive 561 Yellow (CL561Y) Ex561, Em570-630
* ChromaLive 488 Red (CL488R) Ex488, Em630-750

04h Live - BR00122249__2023-03-22T20_03_43-Measurement1 - contains images from 4 fields
24h Live - BR00122249__2023-03-23T14_50_32-Measurement2 - contains images from 4 fields 
48h Live - BR00122249__2023-03-24T12_14_34-Measurement3 - contains images from 1 field

**Batch9:**
Batch 9 was added at a later point in 2025. 

`BR00122244__2023-04-01T13_12_51-Measurement1` plate contains unstained, fixed plate. For detecting compound autofluorescence. All wells, 9 fields.

**Comment by ErinWeisbart** [comment]
The  information above is not sufficient for channel mapping.
TL;DR Batches 1, 2, 3, 6, and 8 can be deduced. We need clarification on Batches 4, 5, and 7

**Batch 1**, **Batch 2**, and **Batch 3** have 8 channels imaged - the 6 channels mentioned above, along with Brightfield, and Phase Contrast (according to the load_data.csv).

**Batch 4** has 7 channels imaged. There are no load_data.csv in CPG so I do not know how to map them. (It does not look like they were analyzed)

**Batch 5** has 7 channels imaged.
In the load_data.csv currently CPG they are mapped:
Phase -1
Brightfield - 2
CL488R - 3
CL488Y - 4
CL561YE - 5
647 - 6
DNA - 7

I can confirm by eye that 1 is phase, 2 is Brightfield, and 7 is DNA. We need to know what 647 is. In batches 1-3 it's mito, but these channel 6 images do not look like mitochondria (however channel 5 does, though it is mapped to CL561YE).

**Batch 6** has 6 channels imaged - the 5 channels mentioned above and Brightfield

**Batch 7** has 6 channels imaged but there are no load_data.csv in CPG so I do not know how to map them. (It does not look like they were analyzed)

**Batch 8** has 6 channels imaged. The load_data.csv only includes 5 channels which are the 3 listed above and Brightfield and Phase so whatever is in the unmapped 6th channel should presumably be ignored.

**Reply by sugan89** [reply]
**Batch 4 and 7:**
Yes, the batches 4 and 7 were not analysed since they had only the DRAQ7 and Cas3/7 hence there is no load_data.csv. Based on the information shared from CDoT, the other channels were not stained for anything (as indicated in the legend in the screenshot below). Please find the screenshot of the slide from CDoT. I have also attached a link to the [slide deck](https://docs.google.com/presentation/d/1g_eIqFrbqI38U2Av4vXQdKUAMh2tw9VS/edit?usp=drive_link&ouid=108208560698528889368&rtpof=true&sd=true). 
<img width="1808" height="942" alt="image" src="https://github.com/user-attachments/assets/b917fa98-18f0-4bbc-90fc-d993147f944c" />

**Batch 5:**
For the batch 5, channel 647 was not stained with any particular dye and they were excluded from the backends CSV, (the plate BR00122249 was stained only with Saguaro dye )
<img width="2118" height="1190" alt="image" src="https://github.com/user-attachments/assets/dcff8539-f3db-49a8-bd27-c1f4382d6463" />


CL561YE channel of Saguaro dye is similar to mitochondria. CDoT have also observed that channel CL561YE looks like mitochondria and hence they have been mapped so. 
<img width="1796" height="968" alt="image" src="https://github.com/user-attachments/assets/28bf3f6b-277b-4b6d-917c-2b2e8e0191bc" />
